specification/cognitiveservices/data-plane/Speech/VideoTranslation/preview/2024-05-20-preview/video-translation.json

```diff
@@ -0,0 +1,963 @@
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "TranslationCreationWebvttFileFormat",
+        "modelAsString": true
+      }
+    },
+    "VideoTranslationVoiceKind": {
+      "description": "Translation voice kind.",
+      "enum": [
+        "PlatformVoice",
+        "PersonalVoice"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "VideoTranslationVoiceKind",
+        "modelAsString": true
+      }
+    },
+    "PaginatedTranslations": {
+      "description": "Paginated translation list",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Translation list",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Translation"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page.",
+          "format": "uri",
+          "type": "string"
+        }
+      }
+    },
+    "PaginatedIterations": {
+      "description": "Paginated iteration list",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Iteration list",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Iteration"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page.",
+          "format": "uri",
+          "type": "string"
+        }
+      }
+    },
+    "WebvttFile": {
+      "description": "Translation webvtt file input.",
+      "required": [
+        "webvttFileUrl",
+        "webvttFileFormat"
+      ],
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "Translation webvtt file url",
+          "type": "string",
+          "format": "uri"
+        },
+        "format": {
+          "$ref": "#/definitions/TranslationCreationWebvttFileFormat"
+        }
+      }
+    },
+    "Translation": {
+      "description": "Translation response.",
+      "required": [
+        "id",
+        "sourceLocale",
+        "targetLocale",
+        "voiceKind"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/ResourceId"
+        },
+        "description": {
+          "description": "Translation description",
+          "type": "string"
+        },
+        "input": {
+          "$ref": "#/definitions/TranslationInput"
+        },
+        "initialIterationInput": {
+          "$ref": "#/definitions/IterationInput"
+        },
+        "createdDateTime": {
+          "description": "The timestamp when the object was created. The timestamp is encoded as ISO 8601 date and time format (\"YYYY-MM-DDThh:mm:ssZ\", see https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "translationResult": {
```

Comment:
translationResult, latestInputWebvttFile, latestTranslationResult: this part mismatch with examples.
Should be: latestIteration, latestCompletedIteration
If you send PR in azure-rest-api-specs, pipeline will auto check such issue.